pkg/kube/kube.go

```diff
@@ -99,6 +103,23 @@ func (k *KubeClient) GetClusterStatus() ClusterStatus {
 	return Unknown
 }
 
+func (k *KubeClient) GetApplication(name string) (*wego.Application, error) {
+	cmd := []string{"get", "app", name, "-o", "json"}
+	o, err := k.runKubectlCmd(cmd)
+
+	if err != nil {
+		return nil, fmt.Errorf("could not run kubectl command: %s", err)
+	}
+
+	a := wego.Application{}
+
+	if err := json.Unmarshal(o, &a); err != nil {
+		return nil, fmt.Errorf("could not unmarshal json: %s", err)
```

Comment:
probably here too

Reply:
Same as my previous comment; the `Unmarshal` call doesn't really have anything to do with the `app` name. It might fail no matter which `app` name got passed in.